bundles/tools.vitruv.applications.demo.insurancefamilies/src/tools/vitruv/applications/demo/insurancefamilies/insurance2families/InsuranceToFamiliesHelper.xtend

```diff
@@ -0,0 +1,151 @@
+package tools.vitruv.applications.demo.insurancefamilies.insurance2families;
+
+import edu.kit.ipd.sdq.activextendannotations.Utility
+import edu.kit.ipd.sdq.metamodels.insurance.InsuranceClient
+import edu.kit.ipd.sdq.metamodels.insurance.InsuranceDatabase
+import edu.kit.ipd.sdq.metamodels.families.Family
+import java.util.Collection
+import java.util.ArrayList
+import edu.kit.ipd.sdq.metamodels.insurance.Gender
+import tools.vitruv.change.interaction.UserInteractor
+import tools.vitruv.change.interaction.UserInteractionOptions.WindowModality
+
+enum PositionPreference {
+	Parent,
+	Child
+}
+
+@Utility
+class InsuranceToFamiliesHelper {
+	public final static String EXCEPTION_MESSAGE_FIRSTNAME_NULL = "A insurance clients's name is not allowed to be null."
+	public final static String EXCEPTION_MESSAGE_FIRSTNAME_WHITESPACE = "A insurance clients's name has to contain at least one non-whitespace character."
+	public final static String EXCEPTION_MESSAGE_FIRSTNAME_ESCAPES = "A insurance clients's name cannot contain any whitespace escape sequences."
+	
+	def static InsuranceDatabase getInsuranceDatabase(InsuranceClient insuranceClient) {
+		return if (insuranceClient.eContainer instanceof InsuranceDatabase)	insuranceClient.eContainer as InsuranceDatabase else null
+	}
+	
+	def static String getLastName(InsuranceClient insuranceClient) {
+		insuranceClient.name.split(" ").last
+	}
+	
+	def static String getFirstName(InsuranceClient insuranceClient) {
+		insuranceClient.name.split(" ").get(0)
+	}
+	
+	def static (Family)=>boolean sameLastName(InsuranceClient insuranceClient) {
+		val String newPersonLastname = insuranceClient.lastName
+		return [family|family.lastName.equals(newPersonLastname)]
+	}
+	
+	def static (Family)=>boolean noParent(InsuranceClient insuranceClient) {
+		return [family|if(insuranceClient.gender === Gender.MALE) family.father === null else family.mother === null]
+	}
+	
+	def static void informUserAboutReplacementOfClient(UserInteractor userInteractor, InsuranceClient insuranceClient, Family oldFamily){
+		var StringBuilder message = new StringBuilder()
+			.append("Insurance Client ")
+			.append(insuranceClient.name)
+			.append(" has been replaced by another insurance client in his family (")
+			.append(oldFamily.lastName)
+			.append("). Please decide in which family and role ")
+			.append(insuranceClient.name)
+			.append(" should be.")
```

Comment:
You could use [template expressions](https://www.eclipse.org/xtend/documentation/203_xtend_expressions.html#templates) here if you want. Also in other instances in this class.